jekyll/_docs/configuration.md

```diff
@@ -227,13 +214,13 @@ machine:
     version: openjdk7
```

Comment:
Not sure how we missed this before but this is setting a Java version not NodeJS.